pkgs/tools/admin/pgadmin/update.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl wget jq yq common-updater-scripts prefetch-yarn-deps yarn-lock-converter
+
+set -eu -o pipefail
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+################################################################
+#         This script will update pgadmin4 in nixpkgs          #
+# Due to recent changes upstream, we will need to convert the  #
+#             `yarn.lock` file back to version 1.              #
+#   This isn't trivially done and relies on 3rd party tools    #
+#       and a hand-written converter (in this script).         #
+# Also, the converter cannot check for `github` repos in the   #
+#   `yarn.lock` file, which this script will add automatically #
+################################################################
+
+
```

Comment:
```suggestion
# This script will update pgadmin4 in nixpkgs.
# Due to recent changes upstream, we will need to convert the `yarn.lock` file back to version 1.
# This isn't trivially done and relies on 3rd party tools and a hand-written converter (in this script).
# Also, the converter cannot check for `github` repos in the `yarn.lock` file, which this script will add automatically
```

Reply:
why remove this formatting? Doesn't it make the script more readable?

Reply:
For me actually no but the script does not need to be fancy and readable, just simple and working